docs/spectator/core/meters/timer.md

```diff
@@ -29,6 +30,127 @@ Long Task Timer:
 
 ![Long Task Timer](../../../images/duration_timer.png)
 
+[Percentile Timer]: percentile-timer.md
+## Querying
+
+!!! Note
+    Timers report summarized statistics about the measurements for a time window
+    including the `totalTime`, `count`, `max` and `totalOfSquares`. If you were to simply query for
+    the name of your timer via
+
+    ```
+    nf.cluster,foo,:eq,name,http.req.latency,:eq,:and
+    ```
+
+    you would get a nonsense value that is the sum of the reported statistics.
+
+When querying the results of a timer, use one of the ASL functions below to generate
```

Comment:
"ASL" might not be familiar to a new user that comes directly here. I think it is better to just say operator or function.